src/core/lv_obj_style.c

```diff
@@ -286,6 +298,9 @@ void lv_obj_refresh_style(lv_obj_t * obj, lv_style_selector_t selector, lv_style
     bool is_ext_draw = lv_style_prop_has_flag(prop, LV_STYLE_PROP_FLAG_EXT_DRAW_UPDATE);
     bool is_inheritable = lv_style_prop_has_flag(prop, LV_STYLE_PROP_FLAG_INHERITABLE);
     bool is_layer_refr = lv_style_prop_has_flag(prop, LV_STYLE_PROP_FLAG_LAYER_UPDATE);
+    bool is_transform_rotation = lv_style_prop_has_flag(prop, LV_STYLE_TRANSFORM_ROTATION);
```

Comment:
`LV_STYLE_TRANSFORM_ROTATION` is not a flag. Did you mean `prop == LV_STYLE_TRANSFORM_ROTATION`?

Reply:
oops!. correct. I upload new patch. however, I got build error in C/C++ CI / Run tests with 64bit build (pull_request). 
let me check furthur

Reply:
I can't find any dependency between 2nd patch and the fail reason below,
could someone help me? 


C/C++ CI / Run tests with 64bit build (pull_request) 

20/92 Test #23: test_libpng ......................***Failed    0.29 sec
/home/runner/work/lvgl/lvgl/tests/src/test_cases/libs/test_libpng.c:73:test_libpng_1:FAIL: Expected 48 to be less than or equal to 32

-----------------------
1 Tests 1 Failures 0 Ignored 
FAIL

      Start 24: test_lodepng
21/92 Test #24: test_lodepng .....................***Failed    0.31 sec
/home/runner/work/lvgl/lvgl/tests/src/test_cases/libs/test_lodepng.c:73:test_lodepng_1:FAIL: Expected 56 to be less than or equal to 40

-----------------------
1 Tests 1 Failures 0 Ignored 
FAIL

Reply:
I'm not sure, but it seems like the problem came from adding the transform_changed variable to the "struct lv_obj_t" structure.